### ISSUE TITLE
Deprecation warning

### ISSUE THREAD

**Issue by pshem** [eclipse/paho.mqtt.python#450]
While running a unit test on code using `paho-mqtt`, I've run into a deprecation warning:
```python
paho/mqtt/publish.py:127: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
  if not isinstance(msgs, collections.Iterable)
```

Since Python 3.8 has been released and 3.9 is next, I think this is worth fixing. I can write the PR later if the maintainers are interested in getting this fixed

**Comment by tirkarthi** [comment]
I have created https://github.com/eclipse/paho.mqtt.python/pull/485 to fix this. Thanks.

**Comment by PierreF** [comment]
Thank for your report and thank @tirkarthi for your PR. This should be solved by PR #497
